### DISCUSSION TITLE
Where to start?

### DISCUSSION THREAD

**Discussion by jennyjanetech** [davidonzo/Threat-Intel#23] [Q&A]
Hello,
Is there a documentation or guide where to start? downloaded this but not sure how to use this repository and tools =( any guide that you can provide would be great!

**Comment by davidonzo** [comment]
What kind of information do you need?
IoCs are shared in various format. Infos here => https://osint.digitalside.it/

It's available an API client library ready to install and use to query the service => https://github.com/davidonzo/apiosintDS
